src/cswinrt/code_writers.h

```diff
@@ -9415,7 +9415,7 @@ RuntimeHelpers.RunClassConstructor(typeof(%).TypeHandle);
 
 public static IntPtr Make()
 {
-using var marshaler = MarshalInspectable<%>.CreateMarshaler(_factory).As<ABI.WinRT.Interop.IActivationFactory.Vftbl>();
+using var marshaler = MarshalInspectable<%>.CreateMarshaler(_factory).As<ABI.WinRT.Interop.IActivationFactory.Vftbl>(IActivationFactoryMethods.IID);
```

Comment:
While looking into this, I also noticed that the authored component includes a generated `IActivationFactoryVftbl` type, which seems to basically be a duplicate of the vtable types already in WinRT.Runtime anyway? @jlaanstra do you know whether we can just remove (as in, stop generating) that entirely and just switch to the ones in the runtime? Maybe it could be part of your activation factory tweaks?